readthedocs/doc_builder/environments.py

```diff
@@ -301,11 +301,35 @@ def run(self):
 
         self.start_time = datetime.utcnow()
         client = self.build_env.get_client()
+
+        # Create a copy of the environment to update PATH variable
+        environment = self._environment.copy()
+        # Default PATH variable
+        environment[
+            "PATH"
+        ] = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+        # Add asdf extra paths
+        environment["PATH"] += (
+            ":/home/{settings.RTD_DOCKER_USER}/.asdf/shims"
+            ":/home/{settings.RTD_DOCKER_USER}/.asdf/bin"
```

Comment:
```suggestion
            f":/home/{settings.RTD_DOCKER_USER}/.asdf/shims"
            f":/home/{settings.RTD_DOCKER_USER}/.asdf/bin"
```

Reply:
Also, any specific reason we are setting these by default? Doesn't look like we are using them, or at least everything was working without them.